onboard/teleop/src/__main__.py

```diff
@@ -38,6 +39,11 @@ def new_reading(self, reading):
 connection = None
 # front_drone_on = Toggle(False)
 # back_drone_on = Toggle(False)
+electromagnet_toggle = Toggle(False)
+laser_toggle = Toggle(False)
+solenoid_on = Toggle(False)
+electromagnet_on = Toggle(False)
+laser_on = Toggle(False)

```

Comment:
These are left over from '18-'19, we won't need these anymore. If you want to keep them around for an example in case we end up with toggle controlled actuators, you could comment them out.